### DISCUSSION TITLE
xiaomi.kettle.mek01

### DISCUSSION THREAD

**Discussion by IgorMitrofanov** [XiaoMi/ha_xiaomi_home#481] [General]
Hello, I would like to ask you how to turn on the heating on the xiaomi.kettle.mek01 thermopot through your integration?
![Без имени](https://github.com/user-attachments/assets/9549b6e9-6fc9-4b2c-997b-5f98dfcf3881)

**Comment by IgorMitrofanov** [comment]
have "Kettle Stop" but "Kettle Start" is exist...

**Reply by SusanPhevos** [reply]
https://github.com/XiaoMi/ha_xiaomi_home/wiki/Feed,-start-cooking,-play-text-and-execute-text-directive#start-cooking
